### DISCUSSION TITLE
Font Awesome loaded failed with theme's theme.css.

### DISCUSSION THREAD

**Discussion by gitaeron** [Lednerb/bilberry-hugo-theme#448] [Q&A] [answered]
> I'm using latest bilberry-hugo-theme on Windows 10, And everything goes well in the hugo server.
> I use this theme by `Option 2: cloning/copying the theme files`, and I only put these on **github page**, I didn't use **Netlify**.

But then I put **/my-site/public/** to my github page and then **font awesome** just can't loaded with Error > Failed to load resource: the server responded with a status of 404 ().  (Found in Chrome F12 => Console)

check imgurl here: 
- [My github page Error](https://ibb.co/9b18kpn)
- [hugo server](https://ibb.co/Wp8CFbT)

> I tried to use Absolute path like **/fonts/...** or like to put whole url in theme.css, but they're all failed. 
> So I recovered theme.css as original.

I've googled related content and they're just telling me about path problem.. But I'm sure I haven't changed  `\my-site\themes\bilberry-hugo-theme\static\theme.css` and any other thing. 

I just followed the instruction to customized `config.toml` and add some `.md` file in **\my-site\content**.

>My base url in `config.toml` is my github page. 
> I don't know should or shouldn't to put address in here.

You can debug with my github page Repo:   **gitaeron.github.io**
Anything of my page is in that Repo. 
- Branch main is about hugo site
- Branch page is about github page.

I'm not a front developer, I'm just a student who started program by C, and then studying Data Analysis with python.
Please help me :(   Any of help is grateful.

**Comment by igor-baiborodine** [comment]
@gitaeron, let me investigate that.

**Reply by gitaeron** [reply]
> @gitaeron, let me investigate that.

Thank you very much !

**Reply by igor-baiborodine** [reply]
@gitaeron, please approve and merge this pull request: https://github.com/gitaeron/gitaeron.github.io/pull/1

**Comment by gitaeron** [comment, marked as answer]
This issue is about  `.nojekyll` file. 

If you clone bilberry theme, and then use `hugo` to generate static web pages.

# Just don't forget to add `.nojekyll` file to your root path with your site.

**Reply by igor-baiborodine** [reply]
@gitaeron, exactly. I had the same issue before: https://github.com/Lednerb/bilberry-hugo-theme/issues/398
![image](https://user-images.githubusercontent.com/1027701/203627429-45a6923d-df31-4b08-806e-413d7dd029b5.png)

**Reply by gitaeron** [reply]
Oh this is embarrassing that I don't find out that issue.  :|
I suppose it's about `font awesome can't loaded`, but I never searched `icons can't display`.

How annoying about the keyword differ. I learned a lesson about this, I'll change similar keywords on search issues next time. =_=
Anyway, if any noob would search the same way like me, they'll find this discuss.

**Reply by davyvandelaar** [reply]
Hi, I had the issue of no icons on my public page. Adding the '.nojekyll' file in root did the trick. Is it possible that the solution for this is not merged (yet)? I am using Github Pages. As I now know the cause of this and the workaround is working, it is fine for me. Just saying the issue still seems to be there.

**Reply by igor-baiborodine** [reply]
@davyvandelaar, this issue is related only when deployed and hosted on GitHub Pages; therefore, the `.nojekyll` file is not added to the example site. But I added the ["Free Hosting"](https://github.com/Lednerb/bilberry-hugo-theme#free-hosting) section in the README guide where I stressed the addition of the `.nojekyll` file when deployed on GitHub Pages.

![image](https://user-images.githubusercontent.com/1027701/225315359-55ab40f5-c73f-4a6d-abbf-71f61374f547.png)

**Reply by davyvandelaar** [reply]
I totally get that. Thanks for adding the section!
